suites/squid/cephfs/tier-0_cephfs_mirrror_upgrade_seq_6x_to_8x.yaml

```diff
@@ -0,0 +1,263 @@
+            module: cephadm.test_cephadm_upgrade.py
+            polarion-id: CEPH-83574638
+            clusters:
+              ceph1:
+                config:
+                  command: start
+                  service: upgrade
+                  base_cmd_args:
+                    verbose: true
+                  benchmark:
+                    type: rados
+                    pool_per_client: true
+                    pg_num: 128
+                    duration: 10
+                  verify_cluster_health: false
+            destroy-cluster: false
+      desc: Running upgrade, mds Failure and i/o's parallelly
+      abort-on-fail: true
+  - test:
+      abort-on-fail: false
+      desc: Validate the Synchronisation is successful upon upgrade of primary cluster
+      clusters:
+        ceph1:
+          config:
+            name: Validate the Synchronisation is successful upon upgrade of primary cluster
+            clean_up: false
+      module: cephfs_mirror_upgrade.post_upgrade_validate.py
+      name: Validate the Synchronisation is successful upon upgrade.
+      polarion-id: CEPH-83575336
+  - test:
+      name: Upgrade along with IOs
+      module: test_parallel.py
+      clusters:
+        ceph2:
+          config:
+            name: Validate the Synchronisation is successful upon enabling fs mirroring
+      parallel:
+        - test:
+            abort-on-fail: false
+            config:
+              timeout: 30
+              client_upgrade: 1
+              client_upgrade_node: 'node8'
+            desc: Runs IOs in parallel with upgrade process
+            module: cephfs_upgrade.cephfs_io.py
+            name: "creation of Prerequisites for Upgrade"
+            polarion-id: CEPH-83575315
+        - test:
+            name: Upgrade ceph
+            desc: Upgrade cluster to latest version
+            module: cephadm.test_cephadm_upgrade.py
+            polarion-id: CEPH-83574638
+            clusters:
+              ceph2:
+                config:
+                  command: start
+                  service: upgrade
+                  base_cmd_args:
+                    verbose: true
+                  benchmark:
+                    type: rados
+                    pool_per_client: true
+                    pg_num: 128
+                    duration: 10
+                  verify_cluster_health: false
+            destroy-cluster: false
+      desc: Running upgrade, mds Failure and i/o's parallelly
+      abort-on-fail: true
+  - test:
+      abort-on-fail: false
+      desc: Validate the Synchronisation is successful upon upgrade of second cluster
+      clusters:
+        ceph1:
+          config:
+            name: Validate the Synchronisation is successful upon upgrade of second cluster
+            clean_up: true
+      module: cephfs_mirror_upgrade.post_upgrade_validate.py
+      name: Validate the Synchronisation is successful upon upgrade.
+      polarion-id: CEPH-83575336
```

Comment:
Hi Amar,

Can you also include the recent addition [snapdiff tests] to mirror upgrade suite. ?

Reply:
Updated Hemanth. 
Execution in progress